src/server/test/web/obvius/README.md

```diff
@@ -0,0 +1,14 @@
+# Obvius web testing tools
+
+Here are some sample files for testing Obvius routes.
+
+For instance:
+
+```bash
+curl localhost:3000/api/obvius -X POST \
+	-F 'password=password' \
+	-F 'mode=LOGFILEUPLOAD' \
+	-F 'serialnumber=mb-001' \
+	-F 'logfile=@src/server/test/db/web/obvius/mb-001.log.gz'
```

Comment:
```suggestion
	-F 'logfile=@src/server/test/web/obvius/mb-001.log.gz'
```
The directory is off for the location of this file. This is mostly so that anyone working on this is aware of how to run this.